src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/ElementSelector.tsx

```diff
@@ -32,20 +33,24 @@ const ElementSelector = ({
   basePath,
   options,
   value,
+  newValues,
   onChange,
 }: ElementSelectorProps) => {
   return (
     <>
+      Attribute Selector
```

Comment:
This is fairly simple component, but having unit tests will help us catch any potential breakouts in future.